### ISSUE TITLE
Green flavor failing to build raspberry packages

### ISSUE THREAD

**Issue by Itxaka** [rancher/elemental-toolkit#1548]
cpio util is not installed it seems

```
 #11 [ 7/10] RUN rpm2cpio raspberrypi-firmware-dt-2021.03.15-2.1.noarch.rpm | cpio -idmv
 #11 sha256:abda826a890421fe4cd7a1c16f788365b797011d7f7c27380c6cd44328d3612f
 INFO   #11 0.275 /bin/sh: line 1: cpio: command not found
```

**Comment by Itxaka** [comment]
Maybe https://github.com/rancher/elemental-toolkit/pull/1549

**Comment by Itxaka** [comment]
actually fix is on #1551
